i18n/i18n.jsx

```diff
@@ -124,28 +125,34 @@ const languages = {
         order: 15,
         url: uk,
     },
+    ko: {
```

Comment:
I'm not sure if this will fix the error around the missing `fa.json`, but this is actually getting overwritten since it's reusing the same key name as Korean is below
```suggestion
    fa: {
```